javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/ReflectionTypeSolver.java

```diff
@@ -24,12 +24,21 @@
 import java.util.Optional;
 
 /**
+ * Uses reflection to resolve types.
+ * All classes on the classpath used to run your application will be found.
```

Comment:
Still only true if the `boolean` constructor is used and `false` given.
If `true` given or no parameter, then only JRE classes are found.

Reply:
Fixed